frontend/lib/src/components/widgets/DataFrame/hooks/useCustomTheme.ts

```diff
@@ -82,9 +82,9 @@ function useCustomTheme(): Readonly<CustomGridTheme> {
       textHeader: theme.colors.fadedText60,
       textHeaderSelected: theme.colors.white,
       textGroupHeader: theme.colors.fadedText60,
-      headerFontStyle: `${convertRemToPx(theme.fontSizes.sm)}px`,
+      headerFontStyle: `${theme.fontWeights.normal} ${convertRemToPx(theme.fontSizes.sm)}px`,
```

Comment:
Looks like thats the default now, but just confirming we want the header font weight to be normal?